src/containers/Experiment/SamplesTable.js

```diff
@@ -372,55 +381,67 @@ function ScanModal({ sample, scanType }) {
-        et al. Genomics. 2012.{' '}
+        {scanType === PIPELINES.AffymetrixSCAN
+          ? 'SCAN (Single Channel Array Normalization) is a normalization method for single channel (Affymetrix) microarrays that allows us to process individual samples. SCAN models and corrects for the effect of technical bias, such as GC content, using a mixture-modeling approach. For more information about this approach, see the primary publication (Piccolo, et al. Genomics. 2012.'
+          : 'SCAN (Single Channel Array Normalization) is a normalization method for single channel microarrays that allows us to process individual samples. It was originally developed for Affymetrix microarrays. In our system, it has been adapted for Illumina BeadArrays. SCAN models and corrects for the effect of technical bias, such as GC content, using a mixture-modeling approach. For more information about this approach, see the primary publication (Piccolo, et al. Genomics. 2012. '}
         <a
           href="http://doi.org/10.1016/j.ygeno.2012.08.003"
           target="_blank"
           rel="noopener noreferrer"
           className="button button--link"
         >
           DOI: 10.1016/j.ygeno.2012.08.003
-        </a>) and the SCAN.UPC bioconductor package documentation (<a
-          href="http://doi.org/10.1016/j.ygeno.2012.08.003"
-          target="_blank"
-          rel="noopener noreferrer"
-          className="button button--link"
-        >
-          DOI: 10.18129/B9.bioc.SCAN.UPC
-        </a>)
+        </a>){' '}
+        {scanType === PIPELINES.AffymetrixSCAN ? (
+          <span>
+            and the SCAN.UPC bioconductor package documentation (<a
+              href="http://doi.org/10.1016/j.ygeno.2012.08.003"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="button button--link"
+            >
+              DOI: 10.18129/B9.bioc.SCAN.UPC
+            </a>).
+          </span>
+        ) : (
+          '.'
+        )}
       </p>
 
       <h3 className="processing-info-modal__subtitle">Version Information</h3>
       <table>
         <tbody>
-          <tr>
-            <td className="processing-info-modal__version">SCAN</td>
-            <td>0.2</td>
-          </tr>
-          <tr>
-            <td className="processing-info-modal__version">BrainArray</td>
-            <td>0.4.1</td>
-          </tr>
-          <tr>
-            <td className="processing-info-modal__version">Platform Design</td>
-            <td>1.3</td>
-          </tr>
+          {Object.keys(versionInfo).map(packageName => (
+            <tr>
+              <td className="processing-info-modal__version">{packageName}</td>
+              <td>{versionInfo[packageName]}</td>
+            </tr>
+          ))}
         </tbody>
       </table>
     </div>
   );
 }
 
+function _getScanVersionInfo(scanType) {
+  if (scanType === PIPELINES.AffymetrixSCAN) {
+    return {
+      'SCAN.UPC': '2.20.0',
+      Brainarray: '22.0.0'
```

Comment:
Should be `'Brainarray'`, right?

Reply:
Just realized that it is permitted in JavaScript (but illegal in Python). Let's quote `Brainarray` just for consistency.

Reply:
Oops, actually it's more consistent if I remove all the quotes, so I'll do that.

Reply:
Oh, nevermind. Prettier actually formatted those objects automatically, so we should probably go with that.